server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -1779,45 +1779,45 @@ private boolean operateAuthor(AuthorPlan author) throws QueryProcessException {
     try {
       switch (authorType) {
         case UPDATE_USER:
-          authorizer.updateUserPassword(userName, newPassword);
+          IoTDB.authorizer.updateUserPassword(userName, newPassword);
           break;
         case CREATE_USER:
-          authorizer.createUser(userName, password);
+          IoTDB.authorizer.createUser(userName, password);
           break;
         case CREATE_ROLE:
-          authorizer.createRole(roleName);
+          IoTDB.authorizer.createRole(roleName);
           break;
         case DROP_USER:
-          authorizer.deleteUser(userName);
+          IoTDB.authorizer.deleteUser(userName);
           break;
         case DROP_ROLE:
-          authorizer.deleteRole(roleName);
+          IoTDB.authorizer.deleteRole(roleName);
```

Comment:
We need to think about how to combine it with the old BasicAuthorizer. Load through the reflection mechanism. The upper-layer invocation does not need to be triggered.

Reply:
Such an operation has been implemented in Basic Authorizer.get Instance()